mu4/playback/internal/playbackcontroller.cpp

```diff
@@ -139,12 +163,7 @@ void PlaybackController::play()
     }
 
     seek(tick.val);
-
-    bool ok = audioPlayer()->play();
-    if (!ok) {
-        LOGE() << "failed play";
-        return;
-    }
+    sequencer()->play();
```

Comment:
This is a very dangerous code with the current implementation and use of the Sequencer.
Because the methods of the same class instance are called on different threads.
It is difficult to keep track of the fact that there is no simultaneous access to data.
As a result, chaotic crashes are very likely.

Reply:
So, I added
```
    static QTimer timer;
    QObject::connect(&timer, &QTimer::timeout, [this]() {
        static bool isPlay = false;
        if (isPlay) {
            sequencer()->stop();
        } else {
            sequencer()->play();
        }
        isPlay = !isPlay;
    });

    timer.start(10);
```

And stable assert

![image](https://user-images.githubusercontent.com/3818029/99874161-28b39280-2bee-11eb-8d2c-c65f15239455.png)

Reply:
Also another crash! 
Here I wrote about this potential problem 
https://github.com/musescore/MuseScore/pull/6848#discussion_r528035732

![image](https://user-images.githubusercontent.com/3818029/99874270-fc4c4600-2bee-11eb-9827-8b5f057e38fe.png)

Reply:
fixed